src/test/scala/com/typesafe/scalalogging/LoggerWithTaggedArgsSpec.scala

```diff
@@ -46,35 +48,39 @@ class LoggerWithTaggedAargsSpec extends AnyWordSpec with Matchers with Varargs {
       val f = fixture(_.isTraceEnabled)
       import f._
       noException shouldBe thrownBy {
-        logger.trace("This should not throw: {}, {}", arg1, arg2)
+        logger.trace("This should not throw: {}, {} - {}", arg1, arg2, arg3)
       }
+      verify(underlying).trace(any[String], *, *, *)
     }
 
     "not throw ClassCastException when interpolated message is passed" in {
       val f = fixture(_.isTraceEnabled)
       import f._
       noException shouldBe thrownBy {
-        logger.trace(s"This should not throw: $arg1, $arg2")
+        logger.trace(s"This should not throw: $arg1, $arg2, $arg3")
       }
+      verify(underlying).trace(any[String], *, *, *)
     }
   }
 
   "Calling debug with tagged args" should {
 
     "not throw ClassCastException when varargs are passed" in {
-      val f = fixture(_.isTraceEnabled)
+      val f = fixture(_.isDebugEnabled)
```

Comment:
This test and the next were actually not testing anything, as the incorrect log level was being enabled